ios/FluentUI/Tab Bar/TabBarView.swift

```diff
@@ -90,7 +95,7 @@ open class TabBarView: UIView {
         topBorderLine.translatesAutoresizingMaskIntoConstraints = false
         addSubview(topBorderLine)
 
-        heightConstraint = stackView.heightAnchor.constraint(equalToConstant: traitCollection.userInterfaceIdiom == .phone ? Constants.phonePortraitHeight : Constants.padHeight)
+        heightConstraint = stackView.heightAnchor.constraint(equalToConstant: traitCollection.userInterfaceIdiom == .phone ? tokens.phonePortraitHeight : tokens.padHeight)
```

Comment:
I think you can make this a non-optional lazy var and initialize it with the contents like this:

```
private lazy var heightConstraint: NSLayoutConstraint = {
        stackView.heightAnchor.constraint(equalToConstant: traitCollection.userInterfaceIdiom == .phone ? tokens.phonePortraitHeight : tokens.padHeight)
    }()
```

This will avoid the force unwrap (!) in line 99 below.

Reply:
Question for @rdeassis: why does it need the `{ .. }()`? Could that line be more simply written as 
```swift
private lazy var heightConstraint: NSLayoutConstraint = stackView.heightAnchor.constraint(equalToConstant: traitCollection.userInterfaceIdiom == .phone ? tokens.phonePortraitHeight : tokens.padHeight)
```

Reply:
Since it's been initialized at declaration time, we shouldn't need an initialization here as well.